src/app/list/Survey.tsx

```diff
@@ -1,11 +1,106 @@
-import React from 'react';
+import React, { useState } from 'react';
+
+
+interface Question {
+  id: number,
+  content: string,
+  choices: Choice[]
+}
+
+interface Choice {
+  id: number,
+  content: string,
+  type?: string,
+  rate?: number
+}
 
```

Comment:
__Data types should be generalized__
Those data types should be generalized. Because inside `Question.tsx` we have `Question` data type and so is the same for the `Choice` in `Choice.tsx`.